mibidata/mibi_image.py

```diff
@@ -226,9 +226,11 @@ def remove_attr(self, *args):
                 * attempts to remove a required attribute.
                 * an attribute is not defined for this instance.
         """
-        required_rem = [attr for attr in args if attr
+        if isinstance(attributes, str):
+            attributes = list(attributes)
```

Comment:
You need to do `attributes = [attributes]` otherwise it just creates a list of characters in the string passed.